### PR TITLE
Update AD-Forest-Recovery-Resetting-the-krbtgt-password.md

### PR DESCRIPTION
**Pull request by HMR18**
-> [!NOTE]
+ If you intent to reset the Key Distribution Center Service Account password twice. You must wait 10 hours between the password resets.
+> [!NOTE]

### CODE REVIEW

**Comment by PRMerger16** [comment]
@HMR18 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
Still having problems editing the MarkDown source code for the page, I see.

May I help you correct your content, so it becomes mergeable?

**Comment by HMR18** [comment]
Yes of course, please help.

File: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Resetting-the-krbtgt-password.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ Use the following procedure to reset the krbtgt password for the domain. The fol
 
 > [!NOTE]
 > You should perform this operation twice. The password history of the krbtgt account is two, meaning it includes the two most recent passwords. By resetting the password twice you effectively clear any old passwords from the history, so there is no way another DC will replicate with this DC by using an old password.
+-> [!NOTE]
++ If you intent to reset the Key Distribution Center Service Account password twice. You must wait 10 hours between the password resets.
++> [!NOTE]
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Resetting-the-krbtgt-password.md:34]:
```suggestion

> [!NOTE]
> When resetting the Key Distribution Center Service Account password twice, a 10 hour waiting period is required between resets.
```
The formatting is correct, but I also suggest a more technical English phrasing to make it read more official than the previous suggestion. If you find my phrasing to be overly formal and technical, feel free to tell me so.

**Comment by HMR18** [comment]
Hello dear colleague.

Done as requested.

Thank you in advance.

**Comment by illfated** [comment]
Fair enough. I do not see any new change yet, but it is easy to implement: Just go to my [Suggested change] and open the [Commit changes] drop-down box to complete the merge process and have my suggestion implemented. 🙂

**Comment by PRMerger6** [comment]
@eross-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by eross-msft — APPROVED** [review]

**Pull request merged**